src/EditorFeatures/CSharpTest/Squiggles/ErrorSquiggleProducerTests.cs

```diff
@@ -36,32 +37,32 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Squiggles
     [Trait(Traits.Feature, Traits.Features.ErrorSquiggles), Trait(Traits.Feature, Traits.Features.Tagging)]
     public class ErrorSquiggleProducerTests
     {
-        [WpfFact]
-        public async Task ErrorTagGeneratedForError()
+        [WpfTheory, CombinatorialData]
+        public async Task ErrorTagGeneratedForError(bool pull)
```

Comment:
this allows teh same test to test both the pull and push approach.  in nearly all cases they shoudl get the same set of tags.